### ISSUE TITLE
Support modifying HTML links inside a course.

### ISSUE THREAD

**Issue by cycomachead** [beautyjoy/llab-to-edx#24]
This is REALLY hard.

If the content exists, we can probably just modify the URL (assuming we know the page...)

If the content doesn't exist...what do we do?
Can we link to an external webpage, and add `target=_blank` to the URL?

**Comment by xtitter** [comment]
Can you explain the problem a bit more?  Links from inside a page that go to another page?  (Those should probably have specific llab formatting that can be grabbed)

**Comment by cycomachead** [comment]
Yeah, there's a LOT of links in llab to other curriculum pages, and none of them are currently tagged.
